cmd/get_test.go

```diff
@@ -320,7 +320,7 @@ func Test__GetPipeline__Response200(t *testing.T) {
 	RootCmd.Execute()
 
 	if received == false {
-		t.Error("Expected the API to receive GET secrets/aaaaaaa")
+		t.Error("Expected the API to receive GET pipelines/:id")
 	}
```

Comment:
I suggest using the `assert.Equal(t, receiver, true)` assertion lib.

Its included in the CLI project and makes testing a bit more nice in Go.

https://godoc.org/github.com/stretchr/testify/assert

Reply:
It is not exactly equal, more as information to the dev.

Reply:
```
assert.True(t, received, "Expected the API to receive GET pipelines/:id")
```

Reply:
tnx